DataFormats/HcalCalibObjects/data/dosemapHB_4TeV.txt

```diff
@@ -0,0 +1,16 @@
+3.53787e-05 2.58515e-05 1.2848e-05 1.2864e-05 9.64444e-06 6.45818e-06 4.79736e-06 3.41728e-06 2.59323e-06 1.58431e-06 1.30303e-06 7.61866e-07 6.44677e-07 4.22318e-07 3.28374e-07 2.97656e-07 1.3907e-07

```

Comment:
It was originally in `SimG4CMS/Calo`. Here is a direct email quote from you (May 28, 2013):
>from what I see
>
>either CalibFormats/HcalObjects or CalibCalorimetry/HcalAlgos would be ok - but would be a new dependency for SimG4CMS, so maybe best to avoid this solution
>
>So, then potentially good spots are
>DataFormats/HcalCalibObjects (even a good name for it) or
>Geometry/HcalCommonData

Also, during discussions of porting from 62XSLHC release and updating radiation damage models (see e.g. #12513), it was requested to make these classes configurable in Python, hence the `ParameterSet` dependence.